src/query/pipeline/sources/src/processors/sources/input_formats/impls/input_format_csv.rs

```diff
@@ -73,6 +77,25 @@ impl InputFormatTextBase for InputFormatCSV {
         StageFileFormatType::Csv
     }
 
+    fn get_format_settings(settings: &Arc<Settings>) -> Result<FormatSettings> {
+        let timezone = get_time_zone(settings)?;
+        let quote_char = settings.get_quote_char()?.into_bytes();
+        if quote_char.len() != 1 {
+            return Err(ErrorCode::InvalidArgument(
+                "quote_char can only contain one char",
+            ));
+        }
+        Ok(FormatSettings {
+            record_delimiter: settings.get_record_delimiter()?.into_bytes(),
+            field_delimiter: settings.get_field_delimiter()?.into_bytes(),
+            empty_as_default: settings.get_empty_as_default()? > 0,
+            quote_char: quote_char[0],
+            null_bytes: vec![b'\\', b'N'],
```

Comment:
Can you explain a bit? Why `N` is null bytes?

Reply:
just a convention,   https://clickhouse.com/docs/en/operations/settings/settings#format_csv_null_representation

Reply:
currently hard coded, need to be an Format Option too

Reply:
Clickhouse uses `\\N`, maybe we can use the same value?